### PR TITLE
[ISSUE #3299] Add unit tests for org.apache.shenyu.web.filter.HealthF…

### PR DESCRIPTION
**Pull request by wklong**
[ISSUE #3299] Add unit tests for org.apache.shenyu.web.filter.HealthFilter.

### CODE REVIEW

File: shenyu-web/src/test/java/org/apache/shenyu/web/filter/HealthFilterTest.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+@ExtendWith(MockitoExtension.class)
+@MockitoSettings(strictness = Strictness.LENIENT)
+public final class HealthFilterTest {
+
+    private HealthFilter healthFilter;
+
+    private WebFilterChain webFilterChain;
+
+    @BeforeEach
+    void setUp() {
```
Comment (yu199195) [inline shenyu-web/src/test/java/org/apache/shenyu/web/filter/HealthFilterTest.java:52]: add public

File: shenyu-web/src/test/java/org/apache/shenyu/web/filter/HealthFilterTest.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        StepVerifier.create(filter).expectSubscription().verifyComplete();
+    }
+
+    @Test
+    void testFilterNotMatch() {
+        ServerWebExchange webExchange =
+                MockServerWebExchange.from(MockServerHttpRequest
+                        .post("http://localhost:8080/"));
+        Mono<Void> filter = healthFilter.filter(webExchange, webFilterChain);
+        StepVerifier.create(filter).expectSubscription().verifyComplete();
```
Comment (yu199195) [inline shenyu-web/src/test/java/org/apache/shenyu/web/filter/HealthFilterTest.java:75]: match and NotMatch, Is it the same verification method?

File: shenyu-web/src/test/java/org/apache/shenyu/web/filter/HealthFilterTest.java (hunk 1 above)
Comment (wklong) [reply shenyu-web/src/test/java/org/apache/shenyu/web/filter/HealthFilterTest.java:52]: ok

File: shenyu-web/src/test/java/org/apache/shenyu/web/filter/HealthFilterTest.java (hunk 2 above)
Comment (wklong) [reply shenyu-web/src/test/java/org/apache/shenyu/web/filter/HealthFilterTest.java:75]: I think the filter method responds to void, so it's all the same   verification method.
i also see the  org.apache.shenyu.web.filter.FallbackFilterTest

Comment (wklong) [reply shenyu-web/src/test/java/org/apache/shenyu/web/filter/HealthFilterTest.java:75]: Looking forward to giving a good suggestion, I can go to learn it

Comment (yu199195) [reply shenyu-web/src/test/java/org/apache/shenyu/web/filter/HealthFilterTest.java:75]: hi， if match ， maybe verify this result (or method)

```
   protected Mono<Void> doDenyResponse(final ServerWebExchange exchange) {
        String result = JsonUtils.toJson(new Health.Builder().up().build());
        DataBuffer dataBuffer = exchange.getResponse().bufferFactory().wrap(result.getBytes(StandardCharsets.UTF_8));
        return exchange.getResponse().writeWith(Mono.just(dataBuffer));
    }
```

**Review by yu199195 — APPROVED** [review]

**Review by yu199195 — APPROVED** [review]

**Pull request merged**